kalite/coachreports/api_views.py

```diff
@@ -179,11 +188,23 @@ def aggregate_learner_logs(request):
             output_dict["total_in_progress"] += log_objects.filter(complete=False, struggling=False).count()
             output_dict["exercise_attempts"] = AttemptLog.objects.filter(user__in=learners,
                 timestamp__gte=start_date,
-                timestamp__lte=end_date).count()
+                timestamp__lte=end_date, **obj_ids).count()
             if log_objects.aggregate(Avg("streak_progress"))["streak_progress__avg"] is not None:
                 output_dict["exercise_mastery"] = round(log_objects.aggregate(Avg("streak_progress"))["streak_progress__avg"])
         output_logs.extend(log_objects)
         output_dict["total_complete"] += log_objects.filter(complete=True).count()
+
+        object_buffer = LogModel.objects.filter(
+            user__in=learners,
+            latest_activity_timestamp__gte=start_date,
+            latest_activity_timestamp__lte=end_date).values_list(id_field, flat=True)
+
+        if len(object_buffer) > 1:
+            all_object_ids.update(object_buffer)
+        elif len(object_buffer) == 1:
+            all_object_ids.add(object_buffer)
+    if len(all_object_ids) > 0:
+        output_dict["content_parents"] = map(lambda x: {"id": x.get("id"), "title": x.get("title")}, get_content_parents(ids=list(all_object_ids)))
```

Comment:
Probably don't need the if statement here, as the map will just give an empty list.

Also should probably map this to "available_topics".